src/py/mat3ra/made/tools/build/defect/configuration.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional, List, Union, Any
+
+from pydantic import BaseModel
+
+from .enums import PointDefectTypeEnum
+
+
+class BaseDefectConfiguration(BaseModel):
+    # TODO: fix arbitrary_types_allowed error and set Material class type
+    material: Any = None
+    # defect_type type can be an Enum for a specific defect class (for point defect, 2d defect, etc.)
+    defect_type: Union[PointDefectTypeEnum, None] = None
+
+
+class PointDefectConfiguration(BaseDefectConfiguration):
+    defect_type: PointDefectTypeEnum = PointDefectTypeEnum.VACANCY
+    # TODO: should come from Enum of elements
+    specie: Optional[str] = None
```

Comment:
Or from https://github.com/Exabyte-io/periodic-table

Reply:
Ok. We didn't depend on that before